stl/inc/cvt/wbuffer

```diff
@@ -43,7 +43,14 @@ namespace stdext {
             using off_type   = typename _Traits::off_type;
             using state_type = typename _Traits::state_type;
 
-            explicit wbuffer_convert(_Mysb* _Strbuf = nullptr)
+            wbuffer_convert() : _Pcvt(new _Codecvt), _Mystrbuf(nullptr), _Status(_Unused), _Nback(0) {
+                static state_type _State0;

```

Comment:
Pre-existing and only tangentially related: Why are these `static`, could they be `const`, do we really need multiple, do we really need _any_? Why don't we simply value-initialize `_State` rather than copy-assigning it from a zero-initialized `static`?

Reply:
Woops - this is `stdext::cvt::wbuffer_convert`, which I don't care at all about, instead of `std::wbuffer_convert` which is defined in `<xlocbuf>`. Please make the required changes in `<xlocbuf>` as well.

Reply:
Done